README.md

```diff
@@ -1,4 +1,17 @@
-# Google HTTP Client Library for Java
+# Unofficial, unaffiliated and unassociated fork of https://github.com/googleapis/google-http-java-client for internal use in Cryptomator for Android.
+
+> [!IMPORTANT]  
+> This project is provided under the terms of the [Apache License, Version 2.0;](LICENSE) while it is made available to the public,
+> it is important to note that this repository is **not intended for public use** and therefore **no support** is provided whatsoever.
+>
+> For a project designed for use by the public, please refer to the [original repository.](https://github.com/googleapis/google-http-java-client)
+>
+> The README of the original project is reproduced below for your convenience and to provide proper attribution.  
+> This project is not affiliated, associated, endorsed by, or in any way officially connected with Google, or any of its subsidiaries or its affiliates.
+
+---
+
+# Original README
 
 [![Maven][maven-version-image]][maven-version-link]
```

Comment:
Would also remove those three lines

Reply:
[Line 12](https://github.com/cryptomator/google-http-java-client/pull/11/files/adf0aa5d6a558704abb99c890592d95ddf322499..f7865b43e859d527b8b13d44f6c12ddae0fb2f95#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R12) is used to insert a horizontal divider in the rendered README.
The badges declared in [lines 16-18](https://github.com/cryptomator/google-http-java-client/pull/11/files/adf0aa5d6a558704abb99c890592d95ddf322499..f7865b43e859d527b8b13d44f6c12ddae0fb2f95#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R16-R18) have been removed by 95aa6d98df9cd98c7f5ae4a21c7f08e7c8ff65c6.